Userland/Services/DeviceMapper/DeviceEventLoop.cpp

```diff
@@ -216,14 +227,26 @@ ErrorOr<void> DeviceEventLoop::drain_events_from_devctl()
             continue;
 
         if (event.state == DeviceEvent::State::Inserted) {
-            // NOTE: We have a special function for the KCOV device, because we don't
-            // want to create a new MinorNumberAllocationType (e.g. SingleInstance).
-            // Instead, just blindly create that device node and assume we will never
-            // have to worry about it, so we don't need to register that!
-            if (event.major_number == 30 && event.minor_number == 0 && !event.is_block_device) {
-                TRY(create_kcov_device_node());
-                continue;
+            if (!event.is_block_device) {
+                // NOTE: We have a special handling for the pluggable-once devices, etc,
+                // as these device (if they appear) should only "hotplug" (being inserted)
+                // once during the OS runtime.
+                // Therefore, we don't want to create a new MinorNumberAllocationType (e.g. SingleInstance).
```

Comment:
Yeah, it just used a simple `if` statement for the kcov device. but since it is no longer true that only kcov can be inserted this way, I tried to abstract this into some sort of mechanism.